Sources/SmileID/Classes/Camera/CameraManager.swift

```diff
@@ -179,3 +203,15 @@ extension CameraManager: AVCaptureVideoDataOutputSampleBufferDelegate {
         self.sampleBuffer = imageBuffer
     }
 }
+
+extension CameraManager: AVCapturePhotoCaptureDelegate {
+    func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photo: AVCapturePhoto, error: Error?) {
+        guard error == nil else {
+            set(error: .cannotCaptureImage(error!))
+            return
+        }
+        if let imageData = photo.fileDataRepresentation(), let image = UIImage(data: imageData) {
+            self.capturedImage  = image
+        }
```

Comment:
Should we have an else block here?

Reply:
It highly unlikely that the file data representation of the captured photo fails. Notwithstanding I've added an else block that throws an error if this happens.